dts/msm8226/msm8926-huawei-g6-l11.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	model = "Qualcomm MSM 8926 CDP G6-L11";
+	compatible = "huawei,g6-l11", "qcom,msm8926-cdp", "qcom,msm8926", "lk2nd,device";
```

Comment:
Can drop the "qcom,msm8926-cdp", it's not needed

Reply:
Acknowledged.